lib/kubernetes-deploy/runner_task.rb

```diff
@@ -44,11 +45,14 @@ def run!(task_template:, entrypoint:, args:, env_vars: [], verify_result: true)
       else
         record_status_once(pod)
       end
+      ::StatsD.distribution('run.duration', StatsD.duration(start), tags: tags('success'))
       @logger.print_summary(:success)
     rescue DeploymentTimeoutError
+      ::StatsD.distribution('run.duration', StatsD.duration(start), tags: tags('timeout'))
       @logger.print_summary(:timed_out)
       raise
     rescue FatalDeploymentError
+      ::StatsD.distribution('restart.duration', StatsD.duration(start), tags: tags('failure'))
```

Comment:
`run.duration` to match the others

Or maybe we should use `task_runner` or something to be more explicit that it wasn't a deploy or a restart running?